conjure-go-client/httpclient/client_builder.go

```diff
@@ -249,11 +248,11 @@ func newClientBuilderFromRefreshableConfig(ctx context.Context, config Refreshab
 			svc1log.SafeParam("updatedServiceName", s))
 	})
 
-	if tlsConfig, err := subscribeTLSConfigUpdateWarning(ctx, config.Security()); err != nil {
+	tlsConfig, err := newRefreshableTLSConfig(ctx, config.Security())
+	if err != nil {
 		return err
-	} else if tlsConfig != nil {
```

Comment:
Because of the subscription to the security config, this should always return a non-nil refreshable.